openfl/federated/data/sources/pytorch/local_folder.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Dict
+
+from torch.utils.data import Dataset
+
+
+class LabelMapper:
+    def __init__(self):
+        self.label_to_idx = {}
+        self.idx_to_label = {}
```

Comment:
shouldn't there be a way to provide those when initializing the `LabelMapper` object?